scripts/call_contracts.py

```diff
@@ -346,13 +346,11 @@ def whitelist(
 
 
 @cli.command("update-token")
-@click.argument(
-    "token-address", type=str, metavar="ADDRESS", callback=validate_address, help="token address"
-)
-@click.option("--transfer-limit", type=int, help="New transfer limit of token")
-@click.option("--min-lp-fee", type=int, help="New minimum lp fee")
-@click.option("lp-fee-ppm", type=int, help="New lp fee in ppm")
-@click.argument("protocol-fee-ppm", type=int, help="New protocol fee in ppm")
+@click.argument("token-address", type=str, metavar="ADDRESS", callback=validate_address)
+@click.option("--transfer-limit", type=int)
+@click.option("--min-lp-fee", type=int)
+@click.option("lp-fee-ppm", type=int)
+@click.argument("protocol-fee-ppm", type=int)
```

Comment:
What is the reason for removing the `help` arguments from the argument definitions?

Reply:
> What is the reason for removing the `help` arguments from the argument definitions?

Because `click.argument` does not have that argument (heh) and the script fails without even getting to run any of the commands. It simply was never tested with those changes. I did, however, remove the help for `click.option`, which does support `help`, which was wrong. I fixed this now so we only remove it for `click.argument`. Thanks for asking!